src/guides/protocol.md

```diff
@@ -0,0 +1,230 @@
+| `REGISTER`, `UPDATE`, `RENEW`, `TRANSFER`, `FINALIZE` | `UPDATE` | _Update the DNS resource_ |
+| `REGISTER`, `UPDATE`, `RENEW`, `TRANSFER`, `FINALIZE` | `RENEW` | _Renew name ownership before expiration_ |
+| `REGISTER`, `UPDATE`, `RENEW`, `FINALIZE` | `TRANSFER` | _Initiate ownership transfer to new Handshake address_ |
+| `TRANSFER` | `FINALIZE` | _Confirm ownership to new Handshake address (after lockup period)_ |
+| `REGISTER`, `UPDATE`, `RENEW`, `TRANSFER`, `FINALIZE`  | `REVOKE` | _Permanently burn name -- used when a key is compromised_ |
+
+Transfer and revocation actions are not allowed until
+[about two weeks](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/protocol/networks.js#L310)
+after the reveal period ends.
+
+Names must be renewed within
+[two years](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/protocol/networks.js#L250)
+or they become available for new auctions again.
+
+When a name is updated with either `REGISTER` or `UPDATE`, the new resource
+is stored in a
+[temporary data structure in the chain database.](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/blockchain/chaindb.js#L1839)
+Every 36 blocks (about 6 hours)
+[the txn is committed to the urkel tree](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/blockchain/chaindb.js#L1842-L1843).
+
+From the whitepaper:
+
+> Because our tree is implemented as a series of append-only files, a
+> commission interval is required to prevent history bloat, which may otherwise
+> require the user of the software to compact their history regularly.
+
+### Auctions
+
+Name auctions progress through a [series of states based on time (i.e. block depth).](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/covenants/namestate.js#L127-L159)
+The first time a name is seen on chain, it's state is
+[set to the current block height.](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/blockchain/chain.js#L908)
+This height is saved in the database and used as a reference point for the entire
+auction process. It can be [reset](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/covenants/namestate.js#L229)
+after a name expires. Covenant types will be rejected if they are broadcast during the
+wrong phase.
+
+| Blocks since<br>previous state   | Approx. time | Current state     |
+|----------------------------------|--------------|-------------------|
+| (undefined name)                 |              | `CLOSED`          |
+| < 37                             | ~6 hours     | `OPEN`            |
+| < 720                            | ~5 days      | `BIDDING`         |
+| < 1,440                          | ~10 days     | `REVEAL`          |
+| > 0                              |              | `CLOSED`          |
+
```

Comment:
Maybe mention that this is for mainnet and that the values for each of the networks can be found at `lib/protocol/networks.js`

Reply:
This is mentioned at the very top of the article!

Reply:
`all values below describe mainnet parameters.`

Gotcha 👍